content/applications/inventory_and_mrp/inventory/product_management/product_tracking/lots.rst

```diff
@@ -269,16 +322,27 @@ dashboard.
 From here, products with lot numbers assigned to them will be listed by default, and can be expanded
 to show the lot numbers those products have assigned to them.
 
-To group by lots (or serial numbers), begin by removing any filters in the search bar. Then, click
-the :guilabel:`Group By` drop-down, select :guilabel:`Add Custom Group`, and select
-:guilabel:`Lot/Serial Number` from the drop-down menu. Then, click :guilabel:`Apply`.
+To group by lots, begin by removing any filters in the search bar. Then, click the :guilabel:`Group
+By` drop-down, select :guilabel:`Add Custom Group`, and select :guilabel:`Lot/Serial Number` from
+the drop-down menu. Then, click :guilabel:`Apply`.
 
 Doing so displays all existing lots and serial numbers, and can be expanded to show all quantities
 of products with that assigned number.
 
-.. image:: lots/lots-traceability-report.png
+.. image:: lots/group-by-number.png
    :align: center
    :alt: Lots and serial numbers traceability report.
 
+Traceability report
+-------------------
+
+To view a full stock moves report for a lot number, select the lot number line from the
+:guilabel:`Lots/Serial Number` dashboard. On the lot number form, click the :guilabel:`Traceability
+Report` smart button.
```

Comment:
```suggestion
:guilabel:`Lots/Serial Number` dashboard. On the lot number form, click the :guilabel:`Traceability` smart button.
```
i'm seeing the button as just 'Traceability' during my run through with a 16.0 runbot -- might be worth a quick "double check" during your revision